config-templates/redhat/ocs-remote/4.6/README.md

```diff
@@ -0,0 +1,317 @@
+# Red Hat OpenShift Container Storage - Remote Storage
+
+Red Hat OpenShift Container Storage is a software-defined storage that is optimised for container environments. It runs as an operator on OpenShift Container Platform to provide highly integrated and simplified persistent storage management for containers.
+
+The user has to provide the input values to the custom resource OcsCluster while creating the satellite configuration to deploy OCS
+
+## Prerequisites
+In order to deploy OCS, the following prerequisites are required.
+- Remote storage available in block mode
+- Your cluster must have a minimum of 3 nodes that each have a minimum 16CPUs and 64GB RAM.
+- Your cluster should be compatible with the OCS version that you're trying to install.
```

Comment:
```suggestion
In order to deploy OCS, the following prerequisites are required.

- [Create a Satellite location](cloud.ibm.com/docs/satellite?topic=satellite-locations). 
- [Create a Satellite cluster](cloud.ibm.com/docs/satellite?topic=openshift-satellite-clusters). 
  - Your cluster must have a minimum of 3 worker nodes with at least 16CPUs and 64GB RAM per worker node.
  - Your hosts must meet the [Satellite host requirements](https://cloud.ibm.com/docs/satellite?topic=satellite-host-reqs) in addition to having one of the following remote storage configurations.
    * Two raw devices in block mode that have no partitions or formatted file systems. If your devices have no partitions, each node must have 2 free disks. One disk for the OSD and one disk for the MON.
    * Two raw partitions that have no formatted file system. If your raw devices are partitioned, they must have at least 2 partitions per disk, per worker node.
- [Add your Satellite to a cluster group](cloud.ibm.com/docs/satellite?topic=satellite-cluster-config#setup-clusters-satconfig-groups). 
- Your cluster should be compatible with the OCS version that you're trying to install.
```

Reply:
@derekpoindexter, this is not a requirement for the remote template
```
Two raw devices in block mode that have no partitions or formatted file systems. If your devices have no partitions, each node must have 2 free disks. One disk for the OSD and one disk for the MON.
    * Two raw partitions that have no formatted file system. If your raw devices are partitioned, they must have at least 2 partitions per disk, per worker node
    ```